src/tests/resetPassword.test.js

```diff
@@ -0,0 +1,49 @@
+import {use, request, expect } from 'chai';
+import http from 'chai-http';
+import  app from '../app';
+
+use(http);
+
+let token = '';
+let resetPasswordToken;
+
+describe('Testing resetPassword endpoint', () => {
+    before(async () => {
+        const res = await request(app)
+            .post('/user/signup')
+            .send({ 
+                firstname:"herve",
+                lastname:"bardo",
+                telephone:"0782026576",
+                email:"hervebardo@gmail.com",
+                password:"7654321$#9"
+            });
+
+        token =res.body.token;
+    });
+
+    describe('POST /user/resetpassword', () => {
+        it('Should send a link to email', async () => {
+            const res = await request(app)
+                .post('/user/resetpassword')
+                .send({ email : 'hervebardo@gmail.com' });
+            resetPasswordToken = res.body.token;
+            expect(res).to.have.status(200);
+            expect(res.body).to.have.property('Message');
+            expect(res.body.Message).to.match(/Go to this link to rest your password/i);
+        });
+    });
+
+    describe('PUT /user/resetpassword', () => {
+        it('Should update to a new password', async () => {
+            const res = await request(app)
+                .patch(`/user/resetpassword?token=${resetPasswordToken}`)
+                .set('token', token)
```

Comment:
What request header is this? Is it really necessary? that route is not protected. Your test will run well without any such header
Also, consider increasing test coverage, you have only tested successful requests, what happens to requests that must fail?